jmix-dynattr/dynattr-flowui/src/main/resources/io/jmix/dynattrflowui/messages.properties

```diff
@@ -41,10 +41,12 @@ io.jmix.dynattrflowui.view.category/localizationTab.title=Localization
 io.jmix.dynattrflowui.view.category/categoryAttrsGrid.defaultValue=Default value
 io.jmix.dynattrflowui.view.category/categoryAttrsGrid.currentDate=Current date
 io.jmix.dynattrflowui.view.category/categoryAttrsGrid.entityNotFound=Entity not found
-io.jmix.dynattrflowui.view.category/sourceGridTitle=Source attributes 
+io.jmix.dynattrflowui.view.category/sourceGridTitle=Source attributes
 io.jmix.dynattrflowui.view.category/targetColumnGridTitle=Column
 
 # CategoryAttrs Details
+io.jmix.dynattrflowui.view.categoryattr/notUniqueAttributeName=Name "%s" is not unique
```

Comment:
Perhaps, instead of **Name "%s" is not unique** it will be better to write **Attribute this name already exists in the category**. WDYT?

In the text **Name "abc" is not unique** I see the following problems:
* It's not clear where is this attribute is not unique: in the whole database, in this category only
* There is no need to duplicate the field value in the validation message - it is already displayed right in the text field.